remoteworkitem/trackeritem_repository.go

```diff
@@ -129,27 +129,26 @@ func upsert(ctx context.Context, db *gorm.DB, workItem workitem.WorkItem) (*work
 		return nil, errors.WithStack(err)
 	}
 	var resultWorkItem *workitem.WorkItem
+	c := workItem.Fields[workitem.SystemCreator]
+	var creator uuid.UUID
+	if c != nil {
+		if creator, err = uuid.FromString(c.(string)); err != nil {
+			return nil, errors.Wrapf(err, "Failed to convert creator id into a UUID: %s", err.Error())
```

Comment:
Please :), use lowercase for `Failed` as done in https://github.com/xcoulon/almighty-core/blob/9022df3fd0373ffa2a4ef27467f5b5fea7222681/remoteworkitem/trackeritem_repository.go#L82